### PR TITLE
[4.0] MVC not Mvc

### PR DESCRIPTION
**Pull request by brianteeman**
Codestyle

### CODE REVIEW

File: libraries/src/Form/Field/RadioField.php [hunk 1]
```diff
@@ -62,6 +62,8 @@ public function setup(\SimpleXMLElement $element, $value, $group = null)
 		// The layout for Switcher
 		if (!$element['layout'] && strpos(trim($this->class), 'switcher') === 0)
 		{
+
+			$this->renderLabelLayout = '';
```
Comment (wilsonge) [inline libraries/src/Form/Field/RadioField.php]: What's this change doing?

Comment (brianteeman) [reply libraries/src/Form/Field/RadioField.php]: oops shouldnt be there

Comment (brianteeman) [reply libraries/src/Form/Field/RadioField.php]: fixed

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**